mne_bids/mne_bids.py

```diff
@@ -340,6 +357,12 @@ def _sidecar_json(raw, task, manufacturer, fname, kind,
         warn('No line frequency found, defaulting to 50 Hz')
         powerlinefrequency = 50
 
+    # determine whether any channels have to be ignored:
+    n_ignored = len([ch_name for ch_name in
+                     IGNORED_CHANNELS.get(manufacturer, list()) if
+                     ch_name in raw.ch_names])
```

Comment:
isn't this just `len(ignored_indexes)`?

Reply:
yes, but they exist in different scopes... So you can't simply call `len(ignored_indexes)`

Reply:
oops you are right. Ignore this comment then

Reply:
ok, no worries, will push the one change I made re: your other comment below.